src/extensions/default/AutoUpdate/UpdateStatus.js

```diff
@@ -77,7 +77,8 @@ define(function (require, exports, module) {
     function displayProgress(statusObj) {
         statusObj.spans.forEach(function (span) {
             if (span.id === 'percent') {
-                var bgval = 'linear-gradient(to right, #1474BF ' + span.val + ', rgba(0, 0, 0, 0) 0%';
+                var bgColor = $('#status-bar').css('backgroundColor'),
```

Comment:
This need not be done for every function call. This can be moved outside the function as it does not change.